### ISSUE TITLE
TUDataset returns wrong number of attributes for BZR

### ISSUE THREAD

**Issue by gear** [rusty1s/pytorch_geometric#879]
## 🐛 Bug

Due to the logic in `tu_datasets.py::num_node_labels`, this module will return 0 as the number of `node_attributes` if the `node_attributes` sum to zero. While it doesn't happen in some other datasets, I found out it breaks with `BZR` dataset. This dataset has 3 dims node attributes and 53 dims node labels, but the current dataloader return 0 for `num_node_attributes` and 56 for `num_node_features` (loaded with `use_node_attr=True`).

I'm not sure if using `torch.unique` is a better solution in this case, but it works out for me.

## To Reproduce
```python
from torch_geometric.datasets import TUDatasets 
bzr = TUDatasets("/tmp/", "BZR", use_node_attr=True)
print(brz.num_node_attributes)  # 0 -> should be 3
print(brz.num_node_features)  # 56 -> should be 53
print(brz.num_node_labels)  # 56 -> should be 53
```

## Environment

* OS: Linux 5.3.11-arch1-1
* Python version: 3.7.4
* PyTorch version: 1.2.0
* CUDA/cuDNN version: N/A (cpu)
* GCC version: 9.2.0
* Any other relevant information: Environment doesn't matter here.

## Reason
In the current 1.3.2 `torch_geometric` version, file `torch_geometric/datasets/tu_datasets.py`:

```python
    @property
    def num_node_labels(self):
        if self.data.x is None:
            return 0
        for i in range(self.data.x.size(1)):
            if self.data.x[:, i:].sum() == self.data.x.size(0):
                return self.data.x.size(1) - i
        return 0
```
Let j be the correct last index of node attributes, this function only works well when `self.data.x[:, :j+1] != 0`. In case of BZR datasets (or probably any dataset with zero-centered node attributes), this quantity is 0 (0.0001 to be exact), hence i is 0 when the function returns leading to the bug above. 

Please let me know if I did something wrong here.
